app/controllers/sessions_controller.rb

```diff
@@ -1,7 +1,15 @@
 class SessionsController < ApplicationController
   def new; end
 
-  def create; end
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
If you are down casing here, make sure that you are also downcasing the email when it gets stored in the database. 

It may be something you care about, but this could also be a requirement that you decide is not worth implementing. Does it matter if a user tries to use upper cases in their email address?